frame/support/procedural/src/pallet/expand/storage.rs

```diff
@@ -310,6 +311,65 @@ pub fn process_generics(def: &mut Def) -> syn::Result<Vec<ResultOnEmptyStructMet
 	Ok(on_empty_struct_metadata)
 }
 
+fn augment_final_docs(def: &mut Def) {
```

Comment:
@kianenigma this actually prevents `#[deny(missing_docs)]` from correctly working for storage items, since they now always have a default doc comment.  
We could either only do this if there is already a comment, or always emit a warning when there is not doc. WDYT?